ethcore/src/externalities.rs

```diff
@@ -390,8 +394,12 @@ impl<'a, T: 'a, V: 'a, B: 'a> Ext for Externalities<'a, T, V, B>
 		self.depth
 	}
 
-	fn inc_sstore_clears(&mut self) {
-		self.substate.sstore_clears_count = self.substate.sstore_clears_count + U256::one();
+	fn inc_sstore_refund(&mut self, value: U256) {
+		self.substate.sstore_clears_refund = self.substate.sstore_clears_refund + value;
```

Comment:
We're sure it won't overflow -- Refund will never go above gas provided, and gas provided has upper limit of `U256::max_value()`. I think the issue here is that if it actually overflows, that indicates a consensus bug, and I think it would be better to panic instead of silently saturating if that ever happens.

Reply:
It would be a consensus bug only if it's implemented differently in other clients. Always panicking doesn't seem to me like a good idea (would actually prefer the network to split than to kill large part of it, or even the entire network if it's a private Parity-only chain).

Happy with whatever behavior that doesn't cause a panic (i.e. `overflowing_add` or `saturating_add`) - better be safe (and explicit) than sorry.

Reply:
I realized that I was wrong here. Theoretically it's indeed possible for this value to go over `U256::max_value()` (but practically that should never happen. So I'm going to change this to `saturating_add`.